src/components/Book.js

```diff
@@ -19,23 +19,29 @@ const Book = (props) => {
     <div className={styles.book}>
       <section className={styles.col1}>
         <div>
-          <p>{category}</p>
-          <p>{title}</p>
+          <h3>{category}</h3>
+          <h1>{title}</h1>
           <p>{author}</p>
-          <div>
+          <div className={styles.buttons}>
             <button type="button">Comments</button>
+            <hr />
             <button onClick={handleClick} type="button">Remove</button>
+            <hr />
             <button type="button">Edit</button>
           </div>
         </div>
-        <div>
-          <p>{completed}</p>
-          <p>Completed</p>
+        <div className={styles.completed}>
+          <div className={styles.progress} data-progress="" />
+          <div>
+            <h2>{completed}</h2>
+            <h3>Completed</h3>
+          </div>
         </div>
       </section>
+      <hr />
       <section className={styles.col2}>
         <p>CURRENT CHAPTER</p>
-        <p>{currentChapter}</p>
+        <h3>{currentChapter}</h3>
         <button type="button">UPDATE PROGRESS</button>
       </section>
     </div>
```

Comment:
- [ ] Dear @yuvenalmash, Your design is already impressive, but there are some minor improvements to be added to make your design even better. One is that your buttons need some padding from the right and left, also the font color better be a bit darker to look exactly like the Zeplin design. 
### Zeplin Design
![Screenshot 2023-01-28 173758](https://user-images.githubusercontent.com/69077061/215272571-9d59df6a-a291-4255-a59b-a1fe0f4af587.png)

### Your design 
![Screenshot 2023-01-28 173812](https://user-images.githubusercontent.com/69077061/215272573-5c21285b-5022-4cf8-9296-c7b8fccccb86.png)